liquid_node/commands.py

```diff
@@ -211,6 +211,56 @@ def check_system_config():
         'the "vm.max_map_count" kernel parameter is too low, check readme'
 
 
+def start_job(job, hcl):
+    log.info('Starting %s...', job)
+    spec = nomad.parse(hcl)
+    nomad.run(spec)
+    job_checks = {}
+    for service, checks in nomad.get_health_checks(spec):
+        if not checks:
+            log.warn(f'service {service} has no health checks')
+            continue
+        job_checks[service] = checks
+    return job_checks
+
+
+def populate_secrets(vault_secret_keys, core_auth_apps, liquid_job):
```

Comment:
Please add a docstring to this function, because the arguments here are pretty hard to descipher. Explain what steps this function goes through, what all the parameters do. Add examples too, if you would.

Here are some examples on the formatting you could use:
https://www.sphinx-doc.org/en/master/usage/extensions/example_google.html#example-google